README.md

```diff
@@ -1,17 +1,42 @@
 # tic-tac-toe-ruby
 
-Third Ruby Microverse Project By Ángel Cussi &amp; Oscar Russi
+fifth Ruby Microverse Project By Ángel Cussi &amp; Oscar Russi
 
 ![screenshot](./screenshot.png)
 
-# Bubble sort on Ruby 
+# tic-tac-toe-ruby
 
-Third Microverse project for Ruby Curriculum
+fifth Microverse project for Ruby Curriculum
 
 # What it does
 
-- Created executable file to the project
-- Create folders bin and lib
+- tic tac toe the game for 2 player in the terminal mode
+
+# HOW TO PLAY
+
+- Tic tac toe is a game for two players, the game is played in a board of 3 x 3 cells
+- First players use X symbol
```

Comment:
I think it has some grammatical conflicts. Did you mean `First player uses X symbol` ?